updating/preparing_for_updates/updating-cluster-prepare-past-4.18.adoc

```diff
@@ -0,0 +1,123 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="updating-cluster-prepare-past-4.18"]
+= Preparing to update from {product-title} 4.18 to a newer version
+include::_attributes/common-attributes.adoc[]
+:context: updating-cluster-prepare-past-4.18
+
+toc::[]
+
+Learn about some of the specific concerns around {op-system-base} compute machines before attempting to update from {product-title} 4.18 to a newer version.
+
+[id="migrating-workloads-to-different-nodes_{context}"]
+== Migrating workloads off of package-based {op-system-base} worker nodes
+
+With the introduction of {product-title} 4.19, package-based {op-system-base} worker nodes are no longer supported. Attempting to update your cluster while having these nodes up and running will prevent you from successfully completing the update.
+
+You can reschedule pods running on {op-system-base} compute nodes to run on your {op-system} nodes instead by using node selectors.
+
+For example, the following `Node` object has a label for its operating system information, in this case {op-system}:
++
+.Sample `Node` object with {op-system} label
+[source,yaml,subs="+attributes"]
+----
+kind: Node
+apiVersion: v1
+metadata:
+  name: ip-10-0-131-14.ec2.internal
+  selfLink: /api/v1/nodes/ip-10-0-131-14.ec2.internal
+  uid: 7bc2580a-8b8e-11e9-8e01-021ab4174c74
+  resourceVersion: '478704'
+  creationTimestamp: '2019-06-10T14:46:08Z'
+  labels:
+    kubernetes.io/os: linux
+    failure-domain.beta.kubernetes.io/zone: us-east-1a
+    node.openshift.io/os_version: '{product-version}'
+    node-role.kubernetes.io/worker: ''
+    failure-domain.beta.kubernetes.io/region: us-east-1
+    node.openshift.io/os_id: rhcos <1>
+    beta.kubernetes.io/instance-type: m4.large
+    kubernetes.io/hostname: ip-10-0-131-14
+    beta.kubernetes.io/arch: amd64
+#...
+----
+<1> The label identifying the OS running on the node, to match the pod node selector.
+
+Any pods that you want to schedule to new {op-system} nodes must contain a matching label in its `nodeSelector` field. The following procedure describes how to add it.
+
+.Procedure
+
+. Deschedule the {op-system-base} node currently running your existing pods:
```

Comment:
```suggestion
. Deschedule the {op-system-base} node currently running your existing pods by entering the following command:
```
See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command